lib/tasks/access_tokens.rake

```diff
@@ -0,0 +1,14 @@
+namespace :access_tokens do
+  desc "Add a pre-generated token, using the SHA256 hex digest"
+  task :insert, %i[owner token_digest] => :environment do |_, args|
+    usage_message = "usage: rails access_tokens:insert[<owner>, <token_digest>]".freeze
+    abort usage_message unless args[:owner].present? && args[:token_digest]&.match?(/^[a-f0-9]+$/)
+
+    access_token = AccessToken.create!(
+      description: "inserted with `rails access_tokens:insert`",
+      owner: args[:owner],
+      token_digest: args[:token_digest],
+    )
+    pp(access_token)
```

Comment:
Rough example of how we might want to do it without leaving it down to the dev to generate/use a strong token
```suggestion
    access_token = AccessToken.new(
                                    description: "inserted with `rails access_tokens:insert`",
                                    owner: args[:owner])
    access_token.generate_token
    if access_token.save
      pp(access_token)
    end
```